renv/profiles/dev/renv/settings.json

```diff
@@ -2,11 +2,7 @@
   "bioconductor.version": null,
   "external.libraries": [],
   "ignored.packages": [],
-  "package.dependency.fields": [
-    "Imports",
-    "Depends",
-    "LinkingTo"
-  ],
```

Comment:
This is required or renv will try to snapshot both the profile-specific key in the `DESCRIPTION` file _and_ everything in the `Depends:` key.